src/main/java/seedu/address/model/tag/Subject.java

```diff
@@ -3,32 +3,54 @@
 import static java.util.Objects.requireNonNull;
 import static seedu.address.commons.util.AppUtil.checkArgument;
 
-import java.time.YearMonth;
-import java.time.format.DateTimeFormatter;
-import java.util.Locale;
-
 /**
  * Represents a Subject in the address book.
  * Guarantees: immutable; name is valid as declared in {@link #isValidSubjectName(String)}
  */
 public class Subject {
 
-    public static final String MATHS = "Mathematics";
-    public static final String CS = "Computer Science";
+    /**
+         * Represents all possible Subjects.
+     */
+    private enum Subjects {
+        ENGLISH,
+        CHINESE,
+        ELEMENTARY_MATHEMATICS,
+        ADDITIONAL_MATHEMATICS,
+        PHYSICS,
+        CHEMISTRY,
+        BIOLOGY,
+        GEOGRAPHY,
+        HISTORY,
+        SOCIAL_STUDIES,
+        INVALID
+    }
```

Comment:
I like how you make use of enums to make things more consistent throughout the codebase.